packages/flutter_analysis_options/pubspec.yaml

```diff
@@ -0,0 +1,8 @@
+name: flutter_analysis_options
+author: Flutter Authors <flutter-dev@googlegroups.com>
+description: The base analysis options for the Flutter project
```

Comment:
Try to make the description a bit longer. Ideally 60+ characters. This will help you get the best score on pub.dartlang.org

Reply:
Done.  Added a README as well

Reply:
Actually, I'm not sure we'd ever need/want to publish this to pub.  We should be able to just do something like:

```
  flutter_analysis_options:
    sdk: flutter
```

Although that doesn't help someone who wants to use this outside of Flutter's environment.

Reply:
Actually, we'd need something for the engine, which doesn't depend on Flutter.  In that case I think we would want it on pub.